### ISSUE TITLE
Finding API example no longer works

### ISSUE THREAD

**Issue by BreD1810** [timotheus/ebaysdk-python#307]
I tried the example code:
```python
from ebaysdk.finding import Connection as Finding
from ebaysdk.exception import ConnectionError

try:
    api = Finding(config_file='myfile.yaml')
    response = api.execute('findItemsAdvanced', {'keywords': 'Python'})
    print(response.dict())
except ConnectionError as e:
    print(e)
    print(e.response.dict())
```
I got the result:
`ConnectionResetError: [Errno 104] Connection reset by peer`

The current fix is to change to:
```python
api = Finding(config_file='myfile.yaml', https=True)
```

Related eBay blog post [here](https://developer.ebay.com/news/https-only-finding-api/).

**Comment by timotheus** [comment]
Thanks. I'll update.

**Comment by timotheus** [comment]
After further review, https is set as the default. You shouldn't need to
override this in the constructor unless it's somehow being set to false in
your config file.

What version of the sdk are you using?

**Comment by BreD1810** [comment]
My current version of the sdk is 2.1.5, with Python 3.8.2

My YAML config matches the sample config file provided.

**Comment by fsmosca** [comment]
@timotheus 
```
sdk 2.1.5, Finding, https, False  
current repo code, Finding, https, True
```
Perhaps time to make an official release.

**Comment by timotheus** [comment]
Yes, I will release this week.

**Comment by timotheus** [comment]
fyi, just released 2.2.0 to pypi

**Comment by BreD1810** [comment]
Working great now, thanks!
